client/electron/index.ts

```diff
@@ -471,6 +477,8 @@ function main() {
     // TODO(fortuna): Start the app with the window hidden on auto-start?
     setupWindow();
 
+    await initVpnService();
```

Comment:
You can be more explicit. Like `monitorVpnEvents`. Or you could just inline the function here.

"init" doesn't say much, and it feels like you are doing much more than you actually are. It's also misleading, since some VPN initialization happen somewhere else